libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -114,7 +113,19 @@ const getSharepointData = async (url) => {
   };
 };
 
-const getProjectExcelPath = (url) => url.pathname.replace('.json', '.xlsx');
+const getFilePath = async (ref, repo, owner, excelRef) => {
```

Comment:
I'm a fan of passing in an object if there's more than one or two function parameters.
```suggestion
const getFilePath = async ({ref, repo, owner, excelRef} = {}) => {
```

Reply:
Agree 👍 It's safer to pass an object. Made the change